src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -681,6 +606,20 @@ export class ResultsViewPageStore {
         });
     }
 
+    public set includeGermlineMutations(e: boolean) {
```

Comment:
mabye don't use e because e is kind of reserved for DOM event

Reply:
"include" ?

Reply:
Renamed to 'inlcude'